### PR TITLE
Fix publication date error

### PR DESCRIPTION
**Pull request by lakikowolfe**
Refactor of publication crawler script and related functions to use the R package rentrez instead of easyPubmed.

Solves problem highlighted in issue #10.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -28,4 +28,4 @@ Imports:
 Suggests:
   optparse
```
Comment (avanlinden) [inline DESCRIPTION:29]: need to add `rentrez` -- I think to Suggests? There are other functionalities in this package besides the publication script that don't require rentrez (I don't know if they are ever used, but they exist).

File: R/annotation.R [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+#' @export
+set_up_multiannotations <- function(df, value) {
+  vec <- df[[rlang::as_string(value)]]
+  vec <- stringr::str_split(vec, pattern = ", |,| ,")
+  vec <- purrr::map(vec, trimws)
+  df[[rlang::as_string(value)]] <- vec
+  df
+}
+
+##### lw note: is this function actually ever called? If not we could remove...
```
Comment (avanlinden) [inline R/annotation.R:34]: I think this function was originally provided locally but was replaced in the scripts that use it by the same function that is part of the dccvalidator package (e.g. https://github.com/Sage-Bionetworks/porTools/blob/9c0419c9f6e13d038456cf84e69e52708357f566/inst/scripts/curate-portal-studies-table-pec.R#L50)

I think it's safe to delete here.

**Comment by lakikowolfe** [comment]
@avanlinden is this review done? On my end it says it still requires review. No worries if you're still looking at things. Just wanted to check in.

**Review by avanlinden — APPROVED** [review]

**Comment by avanlinden** [comment]
Looks good to me! Sorry, thought it would re-request review after changes.

**Pull request merged**